views/itineraries-show.jade

```diff
@@ -1,15 +1,15 @@
 extends layout
 
 block content
-
+    if user
```

Comment:
I don't think this line of code does what it is suppose to do. The value of 'user' must be passed in from the server side, and I don't believe there is a value called 'user'.

Reply:
you are right, I needed to create another branch to fix the user validation check